doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -334,21 +332,21 @@ construct supports
 the dependent match annotations:
 
 .. prodn::
-   term += if @term is @pattern as @ident in @pattern return @term then @term else @term
+   term += if @term is @pattern as @ident in @pattern__ind return @term__ret then @term__then else @term__else
 
-As in ``let:``, the variable :token:`ident` (and those in the type pattern)
-are bound in the second :token:`term`; :token:`ident` is also bound in the
-third :token:`term` (but not in the fourth :token:`term`), while the
-variables in the first :token:`pattern` are bound only in the third
-:token:`term`.
+Similarly, this dependent ``if-is-then-else`` construct is just notation for
```

Comment:
```suggestion
This dependent ``if-is-then-else`` construct is just notation for
```

"Similarly" meaning that it is also a notation?  Otherwise, similar to what? (not easy to tell)  Doesn't seem helpful.